frontend/test/playwright/utils/components.ts

```diff
@@ -19,7 +19,11 @@ export const getMenuButton = async (
   page: Page,
   dir: LanguageDirection = "ltr"
 ) => {
-  return page.getByRole("button", { name: t("header.aria.menu", dir) })
+  const menuButton = page.getByRole("button", {
+    name: t("header.aria.menu", dir),
+  })
+  await expect(menuButton).toBeEnabled()
```

Comment:
Ensure that the page is hydrated before we open the pages modal. Otherwise, the tests fail because the event handlers haven't loaded yet.